src/Microsoft.Azure.WebJobs.Extensions.ServiceBus/Listeners/ServiceBusListener.cs

```diff
@@ -79,8 +94,7 @@ public Task StartAsync(CancellationToken cancellationToken)
             }
             else
             {
-                _receiver = _messagingProvider.CreateMessageReceiver(_entityPath, _serviceBusAccount.ConnectionString);
-                _receiver.RegisterMessageHandler(ProcessMessageAsync, _serviceBusOptions.MessageHandlerOptions);
+                _receiver.Value.RegisterMessageHandler(ProcessMessageAsync, _serviceBusOptions.MessageHandlerOptions);
```

Comment:
Can you change these call sites to use the Receiver property? Looks like it was done inconsistently before.